Remotion/Web/Development.WebTesting/BrowserSession/BrowserSessionBase.cs

```diff
@@ -121,13 +121,13 @@ public virtual void Dispose ()
       // Check driver and main browser for null
       if (driverProcess != null)
       {
-        ProcessUtils.GracefulProcessShutdown (driverProcess, c_driverShutdownWaitTime);
+        ProcessUtils.GracefulProcessShutdown (driverProcess, (int) _driverShutdownWaitTime.TotalMilliseconds);
         driverProcess.WaitForExit (60000);

```

Comment:
Why do we have that if we use GracefulProcessShutdown?
Yes, this might be a separate issue.

Reply:
Added [RM-7348](https://re-motion.atlassian.net/browse/RM-7348).